open-api/rest-catalog-open-api.yaml

```diff
@@ -2052,6 +2052,10 @@ components:
           type: boolean
         doc:
           type: string
+        initial-default:
+          type: string
```

Comment:
I don't think the type should be string. The table spec serializes these using the JSON single-value serialization, so it would be `$ref: "#/components/schemas/PrimitiveTypeValue"`.

Reply:
Nice catch, updated